transaction.rb

```diff
@@ -0,0 +1,30 @@
+require_relative "customer"
+require_relative "teller"
+
+class Transaction
+  attr_accessor :transaction_type, :customer, :teller, :transaction_amount, :account_type
+  def initialize(fields)
+    @transaction_type = fields[:type]
+    @customer = fields[:customer]
+    @teller = fields[:teller]
+    @transaction_amount = fields[:amount]
+    @account_type = fields[:account_type]
+  end
+
+  def do_transaction
```

Comment:
With a method name like`do_transaction`, a typical usage reads as follows
```ruby
transaction = Transaction.new(...)
transaction.do_transaction
```
`transaction.do_transaction` reads weird. 

A method names like `do`, `perform`, `execute` would read much better

This method also violates the  *Single Responsibility Principle* transaction message generation shouldn't be happening in this method.
What you can do is set and return some form of status/flag and have a separate method for the transaction message

Reply:
oww okay yeah I will do all the changes what of the issue with the account type in there?